src/Commands/DrushLanguageCommands.php

```diff
@@ -0,0 +1,557 @@
+
+      // Do not re-add existing languages.
+      if (isset($languages[$langcode])) {
+        $this->logger()->warning('The language with code {langcode} already exists.', $messageArgs);
+        continue;
+      }
+
+      // Only allow adding languages for predefined langcodes.
+      // In the foreach loop because the list changes on successful iterations.
+      $predefined = $this->languageManager->getStandardLanguageListWithoutConfigured();
+      if (!isset($predefined[$langcode])) {
+        $this->logger()->warning('Invalid language code {langcode}', $messageArgs);
+        continue;
+      }
+
+      // Add the language definition.
+      $language = ConfigurableLanguage::createFromLangcode($langcode);
+      $language->save();
+
+      // Download and import translations for the newly added language if
+      // interface translation is enabled.
+      if ($this->moduleHandler->moduleExists('locale')) {
+        module_load_include('fetch.inc', 'locale');
+        $options = _locale_translation_default_update_options();
+        if ($batch = locale_translation_batch_update_build([], [$langcode], $options)) {
+          batch_set($batch);
+          $batch =& batch_get();
+          $batch['progressive'] = FALSE;
+
+          // Process the batch.
+          drush_backend_batch_process();
+        }
+      }
+
+      $this->logger()->info('Added language: {langcode}', $messageArgs);
+    }
+  }
+
+  /**
+   * Enable one or more already defined languages.
+   *
+   * @param array $langcodes
+   *   A comma-separated list of langcodes which will be enabled.
+   *
+   * @command language:enable
+   *
+   * @aliases langen,language-enable
+   */
+  public function enable(array $langcodes) {
+    $langcodes = StringUtils::csvToArray($langcodes);
+
+    if (empty($langcodes)) {
+      $this->logger()->error('Please provide one or more comma-separated language codes as arguments.');
+      return;
+    }
+
+    foreach ($langcodes as $langcode) {
+      $messageArgs = ['langcode' => $langcode];
+
+      // In the foreach loop because the list changes on successful iterations.
+      $languages = $this->languageManager->getLanguages();
+
+      // Skip nonexistent languages.
+      if (!isset($languages[$langcode])) {
+        $this->logger()->warning('Specified language does not exist {langcode}', $messageArgs);
+        continue;
+      }
+
+      // Skip already-enabled languages.
+      if ($languages[$langcode]->enabled) {
+        $this->logger()->warning('Language already enabled: !language', $messageArgs);
+        continue;
+      }
+
+      // FIXME find the D8 equivalent: this is D7 logic.
+      db_update('languages')
+        ->condition('language', $langcode)
+        ->fields([
+          'enabled' => 1,
+        ])
+        ->execute();
+
+      // FIXME probably needs a more generic invalidation.
+      // Changing the language settings impacts the interface.
+      $this->cachePage->deleteAll();
```

Comment:
Maybe better to clear all caches? IIRR mostly all caches are depends on language & IMO better to clear them conditionally if any language actually  changed